humanoid_league_rviz_marker/src/humanoid_league_rviz_marker/show_robocup_objects.py

```diff
@@ -126,6 +129,32 @@ def goal_cb(self, msg: GoalRelative):
             self.marker_goal_rel2.color = self.post2_color
             self.marker_publisher.publish(self.marker_goal_rel2)
 
+    def goal_parts_cb(self, msg: GoalPartsRelative):
+        arr = []
+        i = 0
+        for post in msg.posts:
+            post_marker = Marker()
+            pose = Pose()
+            pose.position = post.foot_point
+            post_marker.pose = pose
+            post_marker.type = Marker.CYLINDER
+            post_marker.action = Marker.MODIFY
+            post_marker.id = i
+            post_marker.ns = "rel_goal_parts"
+            color = ColorRGBA()
+            color.r = 1.0
+            color.g = 1.0
+            color.b = 1.0
+            color.a = 1.0
+            post_marker.color = color
+            post_marker.lifetime = rospy.Duration(nsecs=self.goal_lifetime)
+            post_marker.header = msg.header
```

Comment:
I think that the scale of the marker is missing

> scale.x is diameter in x direction, scale.y in y direction, by setting these to different values you get an ellipse instead of a circle. Use scale.z to specify the height.